docs/bastion-mfa.md

```diff
@@ -0,0 +1,19 @@
+## Enabling Multi-factor authentication on the Bastion Host
```

Comment:
Should we put a hyperlink to this in the main readme somewhere? Not sure

Reply:
Added a link to the detailed documentation.